src/providers/nodemodules.ts

```diff
@@ -0,0 +1,45 @@
+import { PackageTarget } from "../install/package.js";
+import { ExactPackage } from "../install/package.js";
+import { Resolver } from "../install/resolver.js";
+// @ts-ignore
+import { fetch } from '#fetch';
+import { JspmError } from "../common/err.js";
+import { importedFrom } from "../common/url.js";
+
+export const name = 'nodemodules';
+
+export function pkgToUrl (pkg: ExactPackage) {
+  return new URL(pkg.version + pkg.name + '/').href;
+}
+
+export function parseUrlPkg (this: Resolver, url: string): ExactPackage | undefined {
+  const nodeModulesIndex = url.lastIndexOf('/node_modules/');
+  if (nodeModulesIndex === -1)
+    return undefined;
+  const version = url.slice(0, nodeModulesIndex + 14);
+  const pkgParts = url.slice(nodeModulesIndex + 14).split('/');
+  const name = pkgParts[0][0] === '@' ? pkgParts[0] + '/' + pkgParts[1] : pkgParts[0];
+  return { registry: 'node_modules', name, version };
+}
+
+async function dirExists (url: URL, parentUrl?: string) {
+  const res = await fetch(url, this.fetchOpts);
```

Comment:
Looks like there is a small bug with paths. When i try to use the `nodemodules` installer in local. It throws 
```shell
 Module not found: file:///node_modules/lit-element imported from file:///Users/<sys-path>/nodemodules-bug/
```

Looks like, it is trying to resolve `node_modules` under `root` like 
```shell
file:///node_modules/lit-element
```

Reply:
Thanks for catching this - was an off by one (would work if you have node_modules a folder below!).